hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieMetadataTableValidator.java

```diff
@@ -226,6 +227,57 @@ public void testSecondaryIndexValidation() throws IOException {
     validateSecondaryIndex();
   }
 
+  @Test
+  public void testFunctionalIndexValidation() throws IOException {
```

Comment:
this test is failing. Could you look into this failure?

Reply:
Addressed